pkg/netpol/connlist/conns_formatter_dot.go

```diff
@@ -16,8 +16,11 @@ import (
 )
 
 const (
-	ipColor        = "red2"
-	nonIPPeerColor = "blue"
+	ipColor         = "red2"
+	nonIPPeerColor  = "blue"
+	edgeWeightLabel = "weight"
```

Comment:
common constans may also be written in `pkg\netpol\internal\dotformatting\dot_output_formatting.go`

Reply:
Done